application/server.py

```diff
@@ -53,6 +53,12 @@ def open_room():
     return render_template("open-room.html")
 
 
+def __repr__(self):
```

Comment:
This needs to go in the model definition for that object inside models.py. Not here. Use the Cell and Piece models to guide you through that.